### PR TITLE
Only use internal pypi when requested

### PR DESCRIPTION
**Pull request by nosmo**
Addresses #1074.

### CODE REVIEW

**Comment by nhandler** [comment]
Can you resolve the conflicts?

**Comment by solarkennedy** [comment]
@nosmo bump

File: yelp_package/Makefile [hunk 1]
```diff
@@ -77,7 +77,7 @@ clean:
 VERSION = $(firstword $(subst -, ,$(RELEASE) ))
 # This sed isn't really right but at least it doesn't crash (and I couldn't get
 # \' or ''' or \\' to work :( )
```
Comment (solarkennedy) [inline yelp_package/Makefile]: Is this comment still relevant?

File: yelp_package/Makefile [hunk 2]
```diff
@@ -77,7 +77,7 @@ clean:
...
-LAST_COMMIT_MSG = $(shell git log -1 --pretty=%B | sed -e "s/'/"/g")
+LAST_COMMIT_MSG = $(shell git log -1 --pretty=%B | sed -e 's/'\''/"/g')
```
Comment (solarkennedy) [inline yelp_package/Makefile]: I'm really surprised this works! How is this second single quote working?

Comment (nosmo) [inline yelp_package/Makefile]: The fact that the string "ends" with the second single quote means that the string concatenation is handled by the shell itself. It works, but it's nasty. I'm going to replace this mess with \x27 which is less immediately readable but less ugly.

**Pull request merged**